external/storm-kafka-client/src/main/java/org/apache/storm/kafka/spout/KafkaPartitionReader.java

```diff
@@ -0,0 +1,10 @@
+package org.apache.storm.kafka.spout;
+
+import org.apache.kafka.clients.consumer.KafkaConsumer;
+import org.apache.kafka.common.TopicPartition;
+
+import java.util.List;
+
+public interface KafkaPartitionReader {
```

Comment:
Nit: Some of these classes could probably go in the internal package, or a new subpackage of the internal package.